splash/qtrender_lua.py

```diff
@@ -487,6 +493,75 @@ def error(error_info):
         ))
         return cmd
 
+    @command(async=True, sets_callback=True, decode_arguments=False, pack_results=True)
+    def with_timeout(self, callback, timeout):
```

Comment:
Maybe we should call it differently everywhere then? Lua manual takes no stance as to how it should be called, naming it just `f`: [f must be a function](https://www.lua.org/manual/5.3/manual.html#pdf-coroutine.create). Maybe we should be a little more expressive about that and call it `func`?